content/downloads.html

```diff
@@ -130,36 +130,36 @@ <h2>Latest CloudStack Releases</h2>
 
 <h3>Source Releases</h3>
-<p>Apache CloudStack's most recent release is 4.15.2.0.
+<p>Apache CloudStack's most recent release is 4.16.0.0.
 </p>
 <p>This is current CloudStack LTS release.</p>
 
 <p>
-    <a href="http://www.apache.org/dyn/closer.lua/cloudstack/releases/4.15.2.0/apache-cloudstack-4.15.2.0-src.tar.bz2">
+    <a href="http://www.apache.org/dyn/closer.lua/cloudstack/releases/4.16.0.0/apache-cloudstack-4.16.0.0-src.tar.bz2">
         <button type="button" class="btn btn-primary btn-md">
-            Get the 4.15.2.0 Source
+            Get the 4.16.0.0 Source
         </button>
     </a>
     <a href="https://downloads.apache.org/cloudstack/KEYS">
         <button type="button" class="btn btn-info btn-xs">
             KEYS
         </button>
     </a>
-    <a href="https://downloads.apache.org/cloudstack/releases/4.15.2.0/apache-cloudstack-4.15.2.0-src.tar.bz2.asc">
+    <a href="https://downloads.apache.org/cloudstack/releases/4.16.0.0/apache-cloudstack-4.16.0.0-src.tar.bz2.asc">
         <button type="button" class="btn btn-info btn-xs">
             PGP
         </button>
     </a>
-    <a href="https://downloads.apache.org/cloudstack/releases/4.15.2.0/apache-cloudstack-4.15.2.0-src.tar.bz2.sha512">
+    <a href="https://downloads.apache.org/cloudstack/releases/4.16.0.0/apache-cloudstack-4.16.0.0-src.tar.bz2.sha512">
         <button type="button" class="btn btn-info btn-xs">
             SHA512
         </button>
     </a>
 </p>
 
-<p>Full release notes can be found in the <a href="http://docs.cloudstack.apache.org/en/4.15.2.0/releasenotes/">version 4.15.2.0 Release Notes</a> website.</p>
+<p>Full release notes can be found in the <a href="http://docs.cloudstack.apache.org/en/4.16.0.0/releasenotes/">version 4.16.0.0 Release Notes</a> website.</p>
 
-<p>Instructions for building from source and installing Apache CloudStack can be found in the <a href="http://docs.cloudstack.apache.org/en/4.15.2.0/installguide/">Installation Guide</a>. Instructions for building from source and upgrading from a previous version of CloudStack to Apache CloudStack 4.15.2.0 can be found in the upgrade section of the Release Notes (see above).</p><br />
+<p>Instructions for building from source and installing Apache CloudStack can be found in the <a href="http://docs.cloudstack.apache.org/en/4.16.0.0/installguide/">Installation Guide</a>. Instructions for building from source and upgrading from a previous version of CloudStack to Apache CloudStack 4.16.0.0 can be found in the upgrade section of the Release Notes (see above).</p><br />
 
 <p>The latest CloudStack LTS maintenance release is 4.14.1.0 as part of the LTS 4.14.x releases.</p>
```

Comment:
@nvazquez we need to replace/remove the old 4.14.1.0 from download page, I'll add a commit.